docs/guides/dev_guide.rst

```diff
@@ -234,7 +241,7 @@ We also do a few things that aren't explicitly called out in PEP 8:
             Nothing at all.
 
         Raises:
-	    CertainQuestionsError: Raises certain questions about the authors sanity.
+      CertainQuestionsError: Raises certain questions about the authors sanity.
```

Comment:
**REQUIRED:** This formatting is not correct

Reply:
These lines were using tab for indent. I replaced 1 tab for 2 spaces and did not notice that. I will fix it, thanks!